gradoop-data-integration/src/main/java/org/gradoop/dataintegration/importer/rdbmsimporter/connection/SQLToBasicTypeMapper.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradoop.dataintegration.importer.rdbmsimporter.connection;
+
+import org.apache.flink.api.common.typeinfo.BasicTypeInfo;
+import org.apache.flink.api.common.typeinfo.TypeInformation;
+import org.gradoop.dataintegration.importer.rdbmsimporter.constants.RdbmsConstants.RdbmsType;
+
+import java.sql.JDBCType;
+
+/**
+ * JDBC type to BasiTypeInfo Mapper
```

Comment:
probably "Basic"? To avoid spelling mistakes you can Str+Space when you are writing a class name. In most IDEs this will insert something like {@link BasicTypeInfo} which enhances your java doc as well since the class is directly linked.

Reply:
Done